src/model/utilities/computerPlayer.test.js

```diff
@@ -0,0 +1,38 @@
+import computerPlayer from './computerPlayer';
+import player from './Player';
+
+describe('Computer Player', () => {
+  const testQuestion = {
+    answers: ["Declan's men and his cook.", 'Los Pollos driver', 'No-Doze', 'Max Arciniega'],
+    questionObject: 'Beaten to death.',
+  };
+
+  it('callback should be called once if we ask the player a question', async () => {
+    const mockCallbackToShowQuestion = jest.fn();
+    const mockCallbackToHandleAnswer = jest.fn();
+    const testComputerPlayer = computerPlayer(player());
+    testComputerPlayer.askQuestion(
+      mockCallbackToShowQuestion,
+      testQuestion,
+      mockCallbackToHandleAnswer,
+    );
+    expect(mockCallbackToShowQuestion.mock.calls).toHaveLength(1);
+    await new Promise((res) => setTimeout(res, 1000));
+    expect(mockCallbackToHandleAnswer.mock.calls).toHaveLength(1);
```

Comment:
zamiast 'toHaveLength', można użyć metody 'toHaveBeenCalledTimes'.
Ja na nią przypadkowo trafiłem ale polecam w tym przypadku :D
```suggestion
    expect(mockCallbackToShowQuestion).toHaveBeenCalledTimes(1);
    await new Promise((res) => setTimeout(res, 1000));
    expect(mockCallbackToHandleAnswer).toHaveBeenCalledTimes(1);
```

Reply:
Ok, poprawię.